editor/src/components/editor/editor-component.tsx

```diff
@@ -129,6 +134,23 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
 
   const onWindowKeyDown = React.useCallback(
     (event: KeyboardEvent) => {
+      if (isFeatureEnabled('Canvas Strategies')) {
+        const existingInteractionSession = editorStoreRef.current.editor.canvas.interactionSession
+        // TODO: we don't handle new interaction started with keyboard now
```

Comment:
Is this / should this be captured in the road map?

Reply:
yep I created an entry for this in the paper doc